new-site/src/components/layout.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Layout component that queries for data
+ * with Gatsby's useStaticQuery component
+ *
+ * See: https://www.gatsbyjs.com/docs/use-static-query/
+ */
+
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { useStaticQuery, graphql } from 'gatsby';
+import { Container } from 'hds-react';
+
+import Header from './header';
+import Footer from './footer';
+import './layout.css';
+
+const Layout = ({ children }) => {
+  const data = useStaticQuery(graphql`
+    query SiteTitleQuery {
+      site {
+        siteMetadata {
+          title
+          footerTitle
+          footerAriaLabel
+          footerCopyrightLinks {
+            label
+            href
+          }
+        }
+      }
+    }
+  `);
+
+  const siteDate = data.site.siteMetadata;
+  const title = siteDate?.title || 'Title';
+  const contentId = 'content';
+
+  return (
+    <div className="page">
+      <Header contentId={contentId} className="pageHeader" siteTitle={title} />
+      <Container id={contentId} className="pageContent" style={{ margin: '0 auto' }}>
+        <main>{children}</main>
+      </Container>
+      <Footer
```

Comment:
I like the abstraction, but it kind of makes it also so that the user who reads this root layout, does not directly notice that the components used here come from hds-react. It also gives an example that the way these components should be used is that you should make abstractions out of them. I wonder if that is what we want?

Reply:
I can remove the abstraction

Reply:
- [ ] remove unnecessary abstraction

Reply:
> Though I guess in reality people end up making those abstractions, like we did in Hauki 😅

I think we used abstractions because we needed to override some styles in common components like buttons so they became hauki-buttons 🤔